cmd/geth/chaincmd.go

```diff
@@ -400,6 +417,25 @@ func importChain(ctx *cli.Context) error {
 	return importErr
 }
 
+func mpsdbUpgrade(ctx *cli.Context) error {
+	stack, _ := makeConfigNode(ctx)
+	defer stack.Close()
+
+	// initialise the tx manager with the dummy tx mgr
+	private.P = &notinuse.DBUpgradePrivateTransactionManager{}
```

Comment:
Why the `notinuse` package?

Reply:
The `DBUpgradePrivateTransactionManager` extends the `PrivateTransactionManager` from the `notinuse` package and only overrides very few methods. When reviewing with Trung we decided that it was not worth putting it in it's own package.